maliput/src/math/mod.rs

```diff
@@ -416,6 +416,91 @@ impl std::fmt::Debug for Quaternion {
     }
 }
 
+/// This class represents the orientation between two arbitrary frames A and D
```

Comment:
Willing to reduce labor, shall we point code documentation to the C++ doxygen code instead of copy-pasting here all the stuff? I think it makes more sense given that it'll be a maintenance burden to have duplicated docs for the bindings. It just simply doesn't scale IMO.

Reply:
Context: 
- With cargo the creation of the documentation is trivial. ([cargo doc](https://doc.rust-lang.org/cargo/commands/cargo-doc.html)). 
- The cargo packages being pushed to crate.io automatically create online documentation at: https://docs.rs/


It is a trade-off at the end but I do see value on having populated documentation on the rust side even though it may require some extra effort for maintenance.